controllers/glanceapi_controller.go

```diff
@@ -446,7 +446,7 @@ func (r *GlanceAPIReconciler) reconcileNormal(ctx context.Context, instance *gla
 	//
 
 	serviceLabels := map[string]string{
-		common.AppSelector: fmt.Sprintf("%s-%s", glance.ServiceName, instance.Spec.APIType),
+		common.AppSelector: instance.Name,
```

Comment:
I think we have a pattern where we are creating across the operators something like:

```
serviceLabels := map[string]string{
		common.AppSelector: glance.ServiceName
		common.ComponentSelector: glance.APIComponent
```

where `APIComponent` should be a `const` like `ServiceName`. 


This significantly improves the ability to search for all resources
owned by an operator, or one of its components:

  oc get all -l service=glance
  oc get all -l component=glance-api

"oc get all" doesn't list Secrets or ConfigMaps, but these can also
be queried.

  oc get secret -l service=glance
  oc get cm -l component=glance-api
 
So I'd like to see something like that instead of using `instance.Name` that can be either internal or external (and it will fail if tomorrow we would like to run an arbitrary numbers of `GlanceAPI` instances, which is in the roadmap).

Reply:
I was using the single label because we have to pass this to `affinity.DistributePods` which only accepts a single label. Probably we have to fix that first to also accept ComponentSelector globally ?

Reply:
Submitted https://github.com/openstack-k8s-operators/lib-common/pull/283